docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.5
```

Comment:
The rebase on top of master wasn't done properly. This file shouldn't exist after https://github.com/weaveworks/scope/pull/2284